src/gui/guiFuncs.cpp

```diff
@@ -307,6 +313,10 @@ void DissolveWindow::updateStatusBar()
         statusLabel_->setText("Running (ESC to stop)");
         statusIndicator_->setPixmap(QPixmap(":/general/icons/play.svg"));
     }
+    else if (!dissolveIterating_ && elapsedTimer_.secondsElapsed() != 0.0)
```

Comment:
I always find exact comparison of `double` values a bit dubious when used in logic statements, so suggesting a greater-than instead.
```suggestion
    else if (!dissolveIterating_ && elapsedTimer_.secondsElapsed() > 0.0)
```